ui/app/templates/vault/cluster/access/method/item/list.hbs

```diff
@@ -0,0 +1,93 @@
+<PageHeader as |p|>
+  <p.top>
+    {{#key-value-header path="vault.cluster.access.methods"}}
+      <li>
+        <span class="sep">
+          /
+        </span>
+        {{#link-to "vault.cluster.access.methods"}}
+          methods
+        {{/link-to}}
+      </li>
+    {{/key-value-header}}
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{capitalize method}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+{{section-tabs model "authShow" paths}}
+<Toolbar>
+  <ToolbarActions>
+    <ToolbarLink
+      @params={{array
+        "vault.cluster.access.method.item.create"
+        (pluralize itemType)
+      }}
+    >
+      Create
+      {{itemType}}
+    </ToolbarLink>
+  </ToolbarActions>
+</Toolbar>
+{{#if model.meta.total}}
+  {{#each model as |item|}}
+    {{#linked-block
+      "vault.cluster.access.method.item.show"
+      (pluralize itemType)
+      item.id
+      class="list-item-row"
+      data-test-method-row=true
+    }}
+      <div class="columns is-mobile">
+        <div class="column is-7-tablet is-10-mobile">
+          {{#link-to
+            "vault.cluster.access.method.item.show"
+            (pluralize itemType)
+            item.id
+            class="is-block has-text-black has-text-weight-semibold"
+            data-test-method-link=true
+          }}
+            {{i-con glyph="role" size=14 class="has-text-grey-light"}}
+            <span class="has-text-weight-semibold">
+              {{item.id}}
+            </span>
+          {{/link-to}}
+          <div class="has-text-grey is-size-8">
+            {{item.id}}
+          </div>
+        </div>
+      </div>
+    {{/linked-block}}
+  {{/each}}
+  {{#if (gt model.meta.lastPage 1)}}
+    {{list-pagination
+      page=model.meta.currentPage
+      lastPage=model.meta.lastPage
+      link="vault.cluster.access.method.item.list"
+    }}
+  {{/if}}
+{{else}}
+  <EmptyState
+    @title="No {{pluralize itemType}} yet"
+    @message="A list of
+      {{pluralize itemType}}
+       in this namespace will be listed here. Create your first
```

Comment:
I don't think we should use "namespace" here

Reply:
Ok yeah I just stole this from identity. Will update this text